README.markdown

```diff
@@ -10,9 +10,17 @@ Test implementation of RESTful API with Dropwizard.
 
 ## Build
 
-Define database connection in `configuration.yaml`.
+Define database connection in `configuration.yaml`:
 
-Compile application into single Java archive:
+To connect to the database, you'll need to download `ojdbc6_g.jar` from Oracle's website.
+
+Link:  http://www.oracle.com/technetwork/database/enterprise-edition/jdbc-112010-090769.html
```

Comment:
use the markdown syntax to make "Oracle's website" a hyperlink to the url supplied.